qa/large-data-tests/src/test/java/org/camunda/bpm/qa/largedata/optimize/OptimizeApiPageSizeTest.java

```diff
@@ -41,13 +41,13 @@
   @ClassRule
   public static ProcessEngineRule processEngineRule = new ProcessEngineRule("camunda.cfg.xml");
 
-
   @BeforeClass
   public static void init() {
     optimizeService = processEngineRule.getProcessEngineConfiguration().getOptimizeService();
 
     // given the generated engine data
-    EngineDataGenerator generator = new EngineDataGenerator(processEngineRule.getProcessEngine(), OPTIMIZE_PAGE_SIZE);
+    // make sure that there are at least two pages of each entity available
+    EngineDataGenerator generator = new EngineDataGenerator(processEngineRule.getProcessEngine(), OPTIMIZE_PAGE_SIZE * 2);
```

Comment:
Wouldn't `OPTIMIZE_PAGE_SIZE * 2` quadruple the amount of entities since the size is already doubled in `EngineDataGenerator`?

Is this intentional?

Reply:
I removed the doubling of the amount in the `EngineDataGenerator` so it can be used for other tests as well. See https://github.com/camunda/camunda-bpm-platform/pull/894/files?file-filters%5B%5D=.java#diff-38e8226d6f10d7399ca8c47f8b0e9d1dR80

Or am I missing something?

Reply:
Sorry, mixed up the diffs. Completely my bad :)